diff -Naur eglibc-2.15/libc/configure b/libc/configure
--- eglibc-2.15/libc/configure	2016-09-08 11:20:06.047961854 +0200
+++ b/libc/configure	2016-09-08 15:24:18.243162976 +0200
@@ -5246,7 +5246,7 @@
   ac_prog_version=`$MAKE --version 2>&1 | sed -n 's/^.*GNU Make[^0-9]*\([0-9][0-9.]*\).*$/\1/p'`
   case $ac_prog_version in
     '') ac_prog_version="v. ?.??, bad"; ac_verc_fail=yes;;
-    3.79* | 3.[89]*)
+    3.79* | 3.[89]* | 4* )
        ac_prog_version="$ac_prog_version, ok"; ac_verc_fail=no;;
     *) ac_prog_version="$ac_prog_version, bad"; ac_verc_fail=yes;;
 
